product-service/db/productList.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "count": 4,
+    "description": "ADVANCED TERMINAL UNIT CONTROLLER - PROGRAMMABLE",
+    "id": "1",
+    "price": 122.4,
+    "title": "ATC1510"
+  },
+  {
+    "count": 6,
+    "description": "Short Product Description3",
+    "id": "2",
+    "price": 10,
+    "title": "ProductNew"
```

Comment:
Please define which types of product in you shop, because right now this is dummy data without any meaning. This was task 1